pfs/lineLists/skyLines.txt

```diff
@@ -19,123 +19,123 @@
 # MERGED=256: Line has been merged into another
```

Comment:
* Why are lines with "wavelength < 630" being flagged as "not visible"? Surely they're visible in the `b` arm?
* How did you determine the 0.05 limit? Is it appropriate to apply this globally?
* Why flag Osterbrock-only lines as `NOT_VISIBLE`? This seems to be overloading `NOT_VISIBLE` yet again. Why are they not suitable for photometry? Wouldn't it be better to add a flag like `NOT_FOR_PHOTOMETRY`?